app/frontend/src/views/form/Preview.vue

```diff
@@ -1,16 +1,15 @@
 <template>
   <BaseSecure :idp="[IDP.IDIR]">
     <h1>
-      PREVIEW
+      {{ $t('trans.preview.preview') }}
```

Comment:
Does this need to be on the right side of the screen? Does the `?` need to be on the other side of the text?